modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/components/sequence/steps/StepSettings.scala

```diff
@@ -192,6 +193,42 @@ object FilterCell {
   def apply(p: Props): Unmounted[Props, Unit, Unit] = component(p)
 }
 
+/**
+ * Component to display the disperser and wavelength
+ */
+object DisperserCell {
+  final case class Props(s: Step, i: Instrument)
+
+  private val component = ScalaComponent.builder[Props]("DisperserCell")
+    .stateless
+    .render_P { p =>
+
+      val nameMapper: Map[String, String] = p.i match {
+        case Instrument.GmosS => enumerations.disperser.GmosSDisperser
+        case Instrument.GmosN => enumerations.disperser.GmosNDisperser
+        case _                => Map.empty
+      }
+
+      val disperser = for {
+        disperser <- instrumentDisperserO.getOption(p.s)
+      } yield nameMapper.getOrElse(disperser, disperser)
+      val centralWavelength = instrumentDisperserLambdaO.getOption(p.s)
+
+      // Formatter
+      val displayedText = (disperser, centralWavelength) match {
+        case (Some(d), Some(w)) => f"$d @ $w%.0f nm"
+        case _                  => "Unknown"
+      }
```

Comment:
Actually, I think now that I'm looking over this that it might need fixing for `Mirror`, since that has no central wavelength and thus `Unknown` will be returned.

I tried to do this by lifting a function `(String, Double) => String` over the `Option`s in a number of ways, e.g.:

`val displayedText = (disperser |@| centralWavelength)((d: String, w: Double) => f"$d @ $w%.0f nm").getOrElse("Unknown")`

However, they all complained about implicits and I couldn't get this to work without importing all of scalaz. This, however, caused issues with operators imported from:

`import japgolly.scalajs.react.vdom.html_<^._`

I'll fix this in another commit for mirror.